eeprom-example.ino

```diff
@@ -0,0 +1,85 @@
+#include <Arduino.h>
+#include <EEPROM.h>
+
+//offset of byte in EEPROM
+#define I2C_OFFSET 0
+//restricted I2C address
+#define I2C_MASTER 0
+
+//counter to show that Arduino is alive
+long cuntr = 0;
```

Comment:
```suggestion
long counter = 0;
```

Reply:
That was intended, but maybe funny only to me :P